doc/related_projects.rst

```diff
@@ -297,11 +244,3 @@ Domain specific packages
 
 - `MSMBuilder <http://msmbuilder.org/>`_  Machine learning for protein
   conformational dynamics time series.
-
-- `scikit-surprise <https://surpriselib.com/>`_ A scikit for building and
-  evaluating recommender systems.
-
-Snippets and tidbits
----------------------
-
-The `wiki <https://github.com/scikit-learn/scikit-learn/wiki/Third-party-projects-and-code-snippets>`_ has more!
```

Comment:
This wiki has not been updated since 2018, I would just delete it after this PR is merged. Having to maintain this list is enough.